test_jpeg/test/test_entropycoder.py

```diff
@@ -0,0 +1,64 @@
+''' testbench for entropy coder '''
+
+from myhdl import block, delay
+from myhdl import instance
+from myhdl import intbv, ResetSignal, Signal, StopSimulation
+from jpegenc.subblocks.RLE.RLECore.entropycoder import entropycoder
+
+# declared constants
+WIDTH = 12
+SIZE = 4
+
+
+@block
+def test():
+
+    '''sequential block for testing entropy coder '''
+
+    clock = Signal(bool(0))
+    reset = ResetSignal(0, active=True, async=True)
+    data_in = Signal(intbv(0)[(WIDTH+1):].signed())
+    size = Signal(intbv(0)[SIZE:])
+    amplitude = Signal(intbv(0)[(WIDTH+1):].signed())
+
+    inst = entropycoder(clock, reset, data_in, size, amplitude)
+
+    @instance
+    def tbclock():
+
+        ''' instance that generates clock '''
```

Comment:
Used `"""` above and `'''` here, I think the other code in test_jpeg use `"""`, use `"""` to be consistent.